### PR TITLE
GCS Website

### PR DESCRIPTION
**Pull request by aravindavk**
Once merged, website will be visible as http://gluster.github.io/gcs

Preview is available here: http://aravindavk.in/gcs

Note: Please do not share the website URL since the content
is not yet final.

Contribution details are updated in README

Signed-off-by: Aravinda VK <avishwan@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+_site
+.sass-cache
+.jekyll-metadata
+vendor
+.bundle
```
Comment (Madhu-1) [inline .gitignore]: need new line

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+_site
+.sass-cache
+.jekyll-metadata
+vendor
```
Comment (Madhu-1) [inline .gitignore]: do we get vendor directory?

File: pages/003-thin-arbiter.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+title: Thin Arbiter Volumes
+---
+```yaml
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: gpv-3
+spec:
+  storageClassName:>
```
Comment (Madhu-1) [inline pages/003-thin-arbiter.md]: storageclassName: glusterfs-csi-thin-arbiter

File: pages/002-block-volumes.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  storageClassName: glustervirtblock-csi
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+```
+
+```bash
+kubectl apply -f pvc-gpv-2.yaml
```
Comment (Madhu-1) [inline pages/002-block-volumes.md]: use create instead of apply in all places

File: .gitignore (hunk 1 above)
Comment (aravindavk) [reply .gitignore]: ok

File: pages/002-block-volumes.md (hunk 4 above)
Comment (aravindavk) [reply pages/002-block-volumes.md]: ok

File: pages/about.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+storage services with container platforms in a manner similar to any
+other container service, and can run on physical, virtual, cloud and
+traditional storage appliances, Gluster Container Storage aka
+GlusterCS, aims to offer a better storage management experience with a
+leaner Gluster stack underneath with focus on helping users to stay
+away from their storage related worries especiallywhen working on a
+hybrid cloud environment.
+
+GlusterCS with a vision of abstracting storage layer as much as
+possible tries to answer the following question
```
[the remark this answers is not in the document]
Comment (aravindavk) [reply pages/about.md]: done

File: pages/about.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+title: About
+layout: page
+permalink: /about/
+---
+
+Software-defined container storage enables organizations to merge
+storage services with container platforms in a manner similar to any
+other container service, and can run on physical, virtual, cloud and
+traditional storage appliances, Gluster Container Storage aka
```
[the remark this answers is not in the document]
Comment (aravindavk) [reply pages/about.md]: done

File: pages/about.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+GlusterCS, aims to offer a better storage management experience with a
+leaner Gluster stack underneath with focus on helping users to stay
+away from their storage related worries especiallywhen working on a
```
[the remark this answers is not in the document]
Comment (aravindavk) [reply pages/about.md]: done

**Comment by nigelbabu** [comment]
I don't see anything that says what is GCS or defines what it is on the website. Shouldn't we add that?

**Comment by Madhu-1** [comment]
@aravindavk for nice representation can we add few lines about GCS to main page

**Comment by Madhu-1** [comment]
for thin arbiter, we need to add some details about new parameters about storage class

currently, this shows an example of creating PVC, do we need to add storage class also?

**Comment by aravindavk** [comment]
> for thin arbiter, we need to add some details about new parameters about storage class
> currently, this shows an example of creating PVC, do we need to add storage class also?

I didn't included storage class step in home page since it may look complex to users. But details will be available in documentation page(separate section for each features). http://aravindavk.in/gcs/doc/thin-arbiter-volumes (SRC: `$SRC/doc/thin-arbiter-volumes.md`)

Documentation and other content needs more discussion. All doc or content input can be provided in markdown format. I feel we can merge this initial PR for fine tuning the content and easy contribution by others.

**Comment by Madhu-1** [comment]
@JohnStrunk PTAL

**Comment by aravindavk** [comment]
> I don't see anything that says what is GCS or defines what it is on the website. Shouldn't we add that?

In commit message?

**Pull request merged**